### PR TITLE
Remove all DIA related code (will be replaced by llvm).

### PR DESCRIPTION
**Pull request by pierricgimmig**
This removes all code that was used to parse pdb files on Windows
(DIA2Dump, DiaManager, etc).  The plan is to use llvm just like we 
do on the Linux version to retrieve debugging information.

b/158003821

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -124,7 +124,6 @@ find_package(Filesystem REQUIRED)
 if(WIN32)
   find_package(minhook REQUIRED)
   find_package(DIASDK REQUIRED)
```
[the remark this answers is not in the document]
Comment (pierricgimmig) [reply CMakeLists.txt:126]: Good points, thanks, all done.

File: OrbitCore/OrbitProcess.cpp [hunk 2]
```diff
@@ -300,29 +300,11 @@ std::shared_ptr<Module> Process::GetModuleFromName(const std::string& a_Name) {
...
-  std::shared_ptr<Module> module = GetModuleFromAddress(a_Address);
-  if (module && module->m_Pdb) {
-    return module->m_Pdb->LineInfoFromAddress(a_Address, o_LineInfo);
-  }
-#else
   UNUSED(a_Address);
   UNUSED(o_LineInfo);
-#endif
-
+  LOG("Dia Loading disabled, TODO: reimplement using LLVM.");
```
Comment (ronaldfw) [inline OrbitCore/OrbitProcess.cpp:307]: TODOs should usually have a bug associated with them. I suggest we also handle this this way. Can you file a bug on and add the bug number here (TODO(b/<number>))?

File: OrbitCore/OrbitType.cpp [hunk 3]
```diff
@@ -18,126 +18,12 @@
...
-    for (const auto& it : m_Hierarchy) {
-      const Parent& parent = it.second;
-      LONG parentOffset = parent.m_BaseOffset;
-      const Type& type = m_Pdb->GetTypeFromId(parent.m_TypeId);
-      type.ListDataMembers(parentOffset, m_DataMembersFull);
-    }
-
-    ListDataMembers(m_BaseOffset, m_DataMembersFull);
-  }
+  LOG("Dia Loading disabled, TODO: reimplement using LLVM.");
```
Comment (ronaldfw) [inline OrbitCore/OrbitType.cpp:26]: Same here, add bug number.

File: OrbitGl/TypesDataView.cpp [hunk 4]
```diff
@@ -213,11 +212,7 @@ void TypesDataView::OnView(const std::vector<int>& a_Items) {
   for (auto& item : a_Items) {
     Type& type = GetType(item);
     std::shared_ptr<Variable> var = type.GetTemplateVariable();
-    var->PrintDetails();
-#ifdef _WIN32
-    std::shared_ptr<OrbitDiaSymbol> diaSymbol = type.GetDiaSymbol();
-    OrbitDia::DiaDump(diaSymbol ? diaSymbol->m_Symbol : nullptr);
-#endif
+    // TODO: output variable info.
```
Comment (ronaldfw) [inline OrbitGl/TypesDataView.cpp:215]: Add a bug number?

**Comment by ronaldfw** [comment]
This should also go into the release/1.48 branch, right?

**Pull request merged**